lib/features.js

```diff
@@ -102,6 +103,30 @@ module.exports = config => {
     },
 
     /**
+     * Return whether or not this request should bypass sign-in confirmation. Currently,
+     * just checks if user has had a verified security event in the past day.
+     *
+     * @param verified
+     * @param recency
+     * @param events
+     * @param request
+     * @returns {boolean}
+     */
+    canBypassSiginConfirmation(verified, recency, events, request) {
```

Comment:
I added those in because I thought there might be case where will eventually do more than just verified and recency of security events. Possibly UA or do some location based estimate on different security events? I can remove and add back when that time comes.